### PR TITLE
chore: Content tweaking

### PR DESCRIPTION
**Pull request by anikbrazeau**
## Small content changes

- SLA spacing and version
- More concise button for link to branching from modal
- Differentiate branching rules and display rules further
- Reorder legend to put main action first
- Consistency in bulleted list formatting
- Autocomplete and autoflow in French legacy terms

### CODE REVIEW

File: app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx [hunk 1]
```diff
@@ -7,18 +7,18 @@ export const Legend = async ({ lang }: { lang: Language }) => {
   const { t } = await serverTranslation("form-builder", { lang });
   return (
     <div className="my-6">
-      <div className="mb-4 flex text-base italic">
-        <span className="inline-block pr-4">
-          <LogicSectionArrowIcon />
-        </span>
-        <span className="inline-block">{t("logic.legend.page")}</span>
-      </div>
       <div className="text-base italic">
```
Comment (timarney) [inline app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx:10]: Looks like `mb-4 flex` was dropped here

We'll need that back to get some spacing

Screenshot 2024-07-31 at 1 26 15 PM

File: app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx [hunk 2]
```diff
@@ -7,18 +7,18 @@ export const Legend = async ({ lang }: { lang: Language }) => {
...
         <span className="inline-block pr-4">
           <LogicOptionArrowIcon className="inline-block" />
         </span>
         <span className="inline-block">{t("logic.legend.option")}</span>
       </div>
+      <div className="mb-4 flex text-base italic">
```
Comment (anikbrazeau) [inline app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx:16]:
```suggestion
      <div className="text-base italic">
```

File: app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx (hunk 1 above)
Comment (anikbrazeau) [inline app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx:10]:
```suggestion
      <div className="mb-4 flex text-base italic">
```

Comment (anikbrazeau) [reply app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/logic/components/flow/Legend.tsx]: Good catch! Thank you!

**Review by timarney — APPROVED** [review]

**Pull request merged**